Disable RDD on Tier3 platforms due to lack of sandboxing and perf degradation.

diff --git modules/libpref/init/StaticPrefList.h modules/libpref/init/StaticPrefList.h
index 7df0b518b267..3b309be1bfce 100644
--- modules/libpref/init/StaticPrefList.h
+++ modules/libpref/init/StaticPrefList.h
@@ -1595,7 +1595,7 @@ VARCACHE_PREF(
 # define PREF_VALUE true
 #elif defined(XP_MACOSX)
 # define PREF_VALUE true
-#elif defined(XP_UNIX)
+#elif defined(XP_LINUX) && !defined(ANDROID)
 # define PREF_VALUE true
 #else
 # define PREF_VALUE false
